airflow-core/src/airflow/ui/src/i18n/locales/he/common.json

```diff
@@ -0,0 +1,79 @@
+{
+  "defaultToGraphView": "ברירת מחדל תצוגת גרף",
+  "defaultToGridView": "ברירת מחדל תצוגת רשת",
+  "logout": "התנתק",
+  "switchToDarkMode": "החלף למצב כהה",
+  "switchToLightMode": "החלף למצב בהיר",
+  "timezone": "איזור זמן",
+  "user": "משתמש",
+  "language": {
+    "chinese": "סינית מסורתית",
+    "english": "אנגלית",
+    "german": "גרמנית",
+    "korean": "קוריאנית",
+    "select": "בחר"
+  },
```

Comment:
Wouldn't it make more sense to write each language in it's own characters? To make it easier for users to get out of an unknown language if selected by mistake.

Reply:
I just merged https://github.com/apache/airflow/pull/51034 - and this introduces this additionally - can you rebase on this? Then I think this comment is implicitly resolved.